other/upgrade-helper.php

```diff
@@ -49,7 +49,7 @@ function getMemberGroups()
 		return $member_groups;
 
 	$request = $smcFunc['db_query']('', '
-		SELECT group_name, id_group
+		SELECT groupName, id_group
```

Comment:
Is this code intended to run prior to 2.0?  groupName will fail for any 2.0 or higher upgrades.

Maybe detect what version we are upgrading from? Or check if the column exists?

Reply:
Correct, this routine is for 1.x only.  That is the problem being fixed here.  The getMemberGroups() function is only called from the upgrade_1-0.sql file.